<?xml version='1.0'?>
<launch>
    <node pkg="multi_uav" type="pid_control" name="pid_controller_node" output="screen" />

</launch>
